src/pages/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import GoogleLogin from 'react-google-login';
+
+const Home = () => {
+const [loginData, setLoginData] = useState(
```

Comment:
as a recommendation you can save the login data in a global context so that you can consult it in the components you need